vsphere/datadog_checks/vsphere/api.py

```diff
@@ -439,9 +440,12 @@ def get_vsan_metrics(self, cluster_nested_elts, entity_ref_ids, id_to_tags, star
                 )
             health_metrics.append(processed_health_metrics)
 
+            redapl_cluster_data = {'name': cluster_reference.name, 'vcenter_server': self.config.hostname}
```

Comment:
Does this redapl data need to be tied to a hostname/host?

Reply:
Yes, hostname is part of resource information that we are reporting to the backend (it will be a part of the resource's row in the backend table).